modules/virt-enable-guest-log-default-cli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/support/virt-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enable-guest-log-default-cli_{context}"]
+= Enabling default access to VM guest system logs with the CLI
+
+You can enable default access to virtual machine (VM) guest system logs by editing the `HyperConverged` custom resource (CR).
```

Comment:
**thought:** I see that both VM and CR are already use on the page before this section, I'm not sure if we still want to define it here. Based on other modules I think we are doing this in all modules.

Reply:
Haha, I usually do it on a per-module basis but it does look excessive with this many short modules in a row. Removing most instances. Thanks :)